app/models/ability.rb

```diff
@@ -48,7 +48,7 @@ def midata_user_permissions(user)
   # rubocop:enable Metrics/MethodLength
 
   def assistant_leader_permission(user)
-    participants = Participant.assistant_leader.where(email: user.email)
+    participants = Participant.where(role: %i[assistant_leader helper], email: user.email)
```

Comment:
by doing that: might we now distinguish between LL/AL and other leaders on the show-page of a unit what to show exactly?